src/HDInsight/HDInsight/ChangeLog.md

```diff
@@ -19,6 +19,11 @@
 -->
 ## Upcoming Release
 
+## Version 6.1.0
+* Added warning message for planning to replace the type of property `DiskEncryption` of type `Microsoft.Azure.Commands.HDInsight.Models.AzureHDInsightCluster` from `Microsoft.Azure.Management.HDInsight.Models.DiskEncryptionProperties` to `Azure.ResourceManager.HDInsight.Models.HDInsightDiskEncryptionProperties`.
```

Comment:
I suggest to add one more sentence here like "This change adds some warning messages for the incoming breaking change, the details are as below:".   It is like summary then concrete items